.github/workflows/prepare-prod-release.yml

```
@@ -0,0 +1,14 @@
+name: Prepare prod release
+on:
+  workflow_dispatch:
+  schedule:
+      - cron: "0 0 * * 1" # Every Monday midnight
+jobs:
+  open-pr:
+    steps:
+      - uses: peter-evans/create-pull-request@v4
```

Freeze the 3rd-party action:
```suggestion
      - uses: peter-evans/create-pull-request@2b011faafdcbc9ceb11414d64d0573f37c774b04  # 4.2.3

---

already removed it but thanks!